### PR TITLE
Fix default settings to be sufficient to run app

### PR DESCRIPTION
**Pull request by vi4m**
RALPH_DEBUG=True will be env setting in deb default installation.
User can change it without writing config file. Default is False.

### CODE REVIEW

**Comment by mkurek** [comment]
i can't agree with this change. can't you use dev settings in deb? Or create new file for deb settings?

**Comment by vi4m** [comment]
It would require to copy all the file contents. I think getenv() variable will be better?

**Comment by mkurek** [comment]
you could use (in `ralph.settings.deb` or sth similar (deb vs dev typo)):
```
from ralph.settings.prod import *
DEBUG = True
```
if you want to use env variable name it other than `DEBUG` (ex. `RALPH_FORCE_DEBUG`)

**Comment by vi4m** [comment]
Why not RALPH_DEBUG ?

**Comment by mkurek** [comment]
`RALPH_DEBUG` is ok too - it just has to be variable which is not-so-easy (™) to overwrite (user has to set this value on purpose, not accidentally)

**Comment by vi4m** [comment]
Ok, this will come with RALPH_DEBUG env setting with the safe default value.

File: src/ralph/settings/prod.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 from ralph.settings import *  # noqa
 
-# FIXME: when going for full production, change it to False
-DEBUG = False
+DEBUG = os.environ.get('RALPH_DEBUG', False)
```
Comment (mkurek) [inline src/ralph/settings/prod.py]: use `os_env_true` (it's shortcut to properly handle boolean env variables)

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**